middleware/2022-10-04-1.3.40.md

```diff
@@ -0,0 +1,49 @@
+---
+authors: poscreator
+slug: middleware/1.3.40
+tags: [Middleware, Germany]
+---
+
+# fiskaltrust.Middleware 1.3.40 (Germany)
+_october 04, 2022_
+
+In this version of the Middleware, we have added a SCU parameter to configure the Heap memory of the FCC, we also have added a queue parameter to configure the migrations timeout, we have modified the DSFINV-K export to exclude non pos-receipts lowering memory consumption, we have created some modi for the TAR-Export and we have fixed bugs regarding failing TAR-Export using a Fiskaly SCU and a bug concerning the indexes creation.
+
```

Comment:
Please add the following snippet after the intro to "cut" the short description:

```suggestion
<!--truncate-->
```